### PR TITLE
News Unpin - 1st doc News app

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
 |image172|
 
 
+Unpinning a News in the home page
+----------------------------------
+
+As a platform-wide publisher (publisher:/platform/users role required), you are allowed to unpin any pinned article from the home page.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: The required role is publisher:/platform/web-contributors.

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
 |image172|
 
 
+Unpinning a News in the home page
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
Unpinning a News from the home page
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+As a platform-wide publisher (publisher:/platform/users role required), you are allowed to unpin any pinned article from the home page. 
+The option unpin allows you to undo the pin action for an article and remove it from the home page block.
+
+As for pin, the unpin function is available at three locations described below:
+
+
+1- Unpinning a News from edit form : 
+
+To unpin an article from the edit form, you have to display the article's details then click on edit icon.
+The edit form is displayed, you only have to untick the checkbox "Pin article"
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
Once the edit form is displayed, you only have to untick the checkbox "Pin article" and validate.
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -2169,4 +2229,11 @@ to display the original timestamp of the comment.
 .. |image179| image:: images/sharenews/confirm_delete.png
 .. |image180| image:: images/sharenews/resume_article.png
 .. |image181| image:: images/sharenews/pin_details.png
+.. |image182| image:: images/sharenews/unpin_activity.png
+.. |image183| image:: images/sharenews/unpin_popup.png
+.. |image184| image:: images/sharenews/unpin_success.png
+.. |image185| image:: images/sharenews/unpin_icon.png
+.. |image186| image:: images/sharenews/unpin_edit.png
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst:2236]: It seems this image is missing.

File: docs/GettingStarted.rst [hunk 5]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+When you choose this option, a confirmation popup appears:
+
+|image183|
+
+After confirmation, a success message appears:
+
+|image184|
+
+
+*3- Unpinning a News from News details : To unpin a News from details, you only need to click on the available unpin icon.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
3- Unpinning a News from News details : to unpin a News from details, you need to click on the unpin icon.
```

File: docs/GettingStarted.rst [hunk 6]
```diff
@@ -1322,6 +1367,21 @@ The content of the related draft will be displayed in the form and you can updat
 
 |image180|
 
+News application
+-----------------
+
+The News application allows you to display all the articles posted in the spaces that you have access to as well as the pinned to home page articles.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
The News application allows you to display all the articles posted in the spaces that you have access to, as well as the articles pinned to home page. 
```

File: docs/GettingStarted.rst [hunk 7]
```diff
@@ -1322,6 +1367,21 @@ The content of the related draft will be displayed in the form and you can updat
 
 |image180|
 
+News application
+-----------------
+
+The News application allows you to display all the articles posted in the spaces that you have access to as well as the pinned to home page articles. 
+To acces to the interface, you have to click on "News" from the left navigation.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
This application is accessible through the link "News" in the left navigation.
```

File: docs/GettingStarted.rst [hunk 8]
```diff
@@ -1322,6 +1367,21 @@ The content of the related draft will be displayed in the form and you can updat
...
+
+|image187|
+
+The page displays the list of "All News". 
+
+|image188|
+
+For each article you will see the title, the illustrative image if inserted, the begining of the news content or the summary, the creation user and its avatar, the publication date and the information about the origin space.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
For each article you can see the title, the illustrative image if inserted, the beginning of the news content or the summary, the creation user and its avatar, the publication date and the information about the origin space.
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 1 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: changed for pin and unpin

File: docs/GettingStarted.rst (hunk 4 above)
Comment (thomasdelhomenie) [reply docs/GettingStarted.rst:2236]: @samehaidi it seems the image is still missing.

File: docs/GettingStarted.rst [hunk 9]
```diff
@@ -1249,7 +1249,7 @@ You can check that the action has been done successfully by connecting to the de
 Pinning a News in the home page
 -------------------------------
 
-As a platform-wide publisher (publisher:/platform/users role required), you are allowed to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
+As a platform-wide publisher (publisher:/platform/web-contributors role required), you are allowed to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
As a platform-wide publisher (publisher:/platform/web-contributors role required), you get permissions to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```

File: docs/GettingStarted.rst [hunk 10]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
 |image172|
 
 
+Unpinning a News from the home page
+----------------------------------
+
+As a platform-wide publisher (publisher:/platform/web-contributors role required), you are allowed to unpin any pinned article from the home page.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
As a platform-wide publisher (publisher:/platform/web-contributors role required), you can unpin any pinned article from the home page. 
```

File: docs/GettingStarted.rst [hunk 11]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
 |image172|
 
 
+Unpinning a News from the home page
+----------------------------------
+
+As a platform-wide publisher (publisher:/platform/web-contributors role required), you are allowed to unpin any pinned article from the home page. 
+The option unpin allows you to undo the pin action for an article and remove it from the home page block.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The unpin option lets you undo the pin action for an article by removing it from the home page news block.
```

File: docs/GettingStarted.rst [hunk 12]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+
+As for pin, the unpin function is available at three locations described below:
+
+
+1- Unpinning a News from edit form : 
+
+To unpin an article from the edit form, you have to display the article's details then click on edit icon.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
To unpin an article from the edit form, display the article's details then click on the ``Edit`` icon.
```

File: docs/GettingStarted.rst [hunk 13]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+Once the edit form is displayed, you only have to untick the checkbox "Pin article" and validate.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Once the edit form is displayed, just untick the checkbox "Pin article" and validate.
```

File: docs/GettingStarted.rst [hunk 14]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+
+|image186|
+
+A confirmation message appears.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
A confirmation message appears : 
```

File: docs/GettingStarted.rst [hunk 15]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+
+|image183|
+
+After confirmation, the article is no longer available in the home page's News block.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After confirmation, the article is no longer available in the home page. 
```

File: docs/GettingStarted.rst [hunk 16]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+|image186|
+
+A confirmation message appears.
+
+|image183|
+
+After confirmation, the article is no longer available in the home page's News block. 
+
+
+2- Unpinning a News from the activity stream : the "unpin article" function is also available from the three-dots menu of the article’s activity.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
2- Unpinning an article from the activity stream : the ``unpin article`` entry is also available from the three-dots menu of the news activity.
```

File: docs/GettingStarted.rst [hunk 17]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+When you choose this option, a confirmation popup appears:
+
+|image183|
+
+After confirmation, a success message appears:
+
+|image184|
+
+
+3- Unpinning a News from News details : to unpin a News from details, you need to click on the unpin icon.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
3- Unpinning an article from news details : to unpin a an article from details, just click on the ``unpin`` icon.
```

File: docs/GettingStarted.rst [hunk 18]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+
+|image185|
+
+The action is successfully done after confirming it as for the two options above. 
+
+You may display the home page to verify that the article is no longer available in the News block.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
You may display the home page to verify that the article is no longer available in the news block.
```

File: docs/GettingStarted.rst [hunk 19]
```diff
@@ -1322,6 +1367,21 @@ The content of the related draft will be displayed in the form and you can updat
 
 |image180|
 
+News application
+-----------------
+
+The News application allows you to display all the articles posted in the spaces that you have access to, as well as the articles pinned to home page.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The News application displays all the articles posted in the spaces that you have access to, as well as the articles pinned to home page. 
```

File: docs/GettingStarted.rst [hunk 20]
```diff
@@ -1322,6 +1367,21 @@ The content of the related draft will be displayed in the form and you can updat
 
 |image180|
 
+News application
+-----------------
+
+The News application allows you to display all the articles posted in the spaces that you have access to, as well as the articles pinned to home page. 
+This application is accessible through the link "News" in the left navigation.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
This application is accessible through the ``News`` link  in the left navigation.
```

File: docs/GettingStarted.rst [hunk 21]
```diff
@@ -1322,6 +1367,21 @@ The content of the related draft will be displayed in the form and you can updat
...
+
+|image187|
+
+The page displays the list of "All News". 
+
+|image188|
+
+For each article you can see the title, the illustrative image if inserted, the beginning of the news content or the summary, the creation user and its avatar, the publication date and the information about the origin space.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
For each article you can see the title, the illustration, the first words of the summary or the article body, the author, the publication date and the information about the source (space).
```

**Review by plamarque — CHANGES_REQUESTED** [review]
First pass with proofreading and simplifications.

**Review by plamarque — APPROVED** [review]
Except for the missing images,  cotent is OK with me.

**Comment by samehaidi** [comment]
Scrrenshot added 
@thomasdelhomenie @plamarque

File: docs/GettingStarted.rst [hunk 22]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
 |image172|
 
 
+Unpinning a News from the home page
+----------------------------------
+
+As a platform-wide publisher (publisher:/platform/web-contributors role required), you can unpin any pinned article from the home page. 
+The unpin option lets you undo the pin action for an article by removing it from the home page news block.
```
Comment (plamarque) [inline docs/GettingStarted.rst:1296]:
```suggestion
The unpin option lets you undo the pin action for an article by removing it from the news block on the home page.
```

File: docs/GettingStarted.rst [hunk 23]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+
+As for pin, the unpin function is available at three locations described below:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
As for pin, the unpin function is available from three locations :
```

File: docs/GettingStarted.rst [hunk 24]
```diff
@@ -1249,7 +1249,7 @@ You can check that the action has been done successfully by connecting to the de
 Pinning a News in the home page
 -------------------------------
 
-As a platform-wide publisher (publisher:/platform/users role required), you are allowed to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
+As a platform-wide publisher (publisher:/platform/web-contributors role required), you get permissions to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
As a platform-wide publisher (publisher:/platform/web-contributors role required), you get permissions to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available from three locations :
```

File: docs/GettingStarted.rst [hunk 25]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+Unpinning a News from the home page
+----------------------------------
+
+As a platform-wide publisher (publisher:/platform/web-contributors role required), you can unpin any pinned article from the home page. 
+The unpin option lets you undo the pin action for an article by removing it from the home page news block.
+
+As for pin, the unpin function is available at three locations described below:
+
+
+1- Unpinning a News from edit form :
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
1- Unpinning a news from edit form : 
```

File: docs/GettingStarted.rst [hunk 26]
```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
...
+To unpin an article from the edit form, display the article's details then click on the ``Edit`` icon.
+Once the edit form is displayed, just untick the checkbox "Pin article" and validate.
+
+|image186|
+
+A confirmation message appears : 
+
+|image183|
+
+After confirmation, the article is no longer available in the home page.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After confirmation, the article is no longer available on the home page. 
```

File: docs/GettingStarted.rst [hunk 27]
```diff
@@ -1322,6 +1367,21 @@ The content of the related draft will be displayed in the form and you can updat
 
 |image180|
 
+News application
+-----------------
+
+The News application displays all the articles posted in the spaces that you have access to, as well as the articles pinned to home page.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The News application displays all the articles posted in the spaces that you have access to, as well as the articles pinned on the home page. 
```

**Review by plamarque — CHANGES_REQUESTED** [review]
minor simplification edits suggested.

**Review by plamarque — APPROVED** [review]
LGTM

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**